Add power usage reporting

bcheck opened the pull request:
Add reporting of power usage.   This is done in the way prescribed in the issue, relaying the ~3 min periodic /report info to an mqtt topic, and also more frequently by changing the ping to issue "/getRuntime" which returns what looks like instantaneous and 30s mean demand.  (or perhaps 5 and 30s?)

The returned power values look to be watts * 4096.  e.g.:
```
20:18:38.144Z [deviceIdHere] recv: map[e:1077 t:b6 uri:/report]
20:18:38.244Z [deviceIdHere] report: {"seconds":182,"watts":1.029052734375}
20:18:38.847Z [deviceIdHere] ping
20:18:39.148Z [deviceIdHere] recv: map[current:NaN meastate:idle power:170b6:1716d relay:open uri:/runtimeInfo voltage:76007:7610a]
20:18:39.249Z [deviceIdHere] report: {"instant":23.04443359375,"avg30s":23.089111328125}
```

My Home Assistant usage: (I'm dropping milliwatts)
```
  - platform: mqtt
    name: voltson_3b_power
    state_topic: "/voltson/[deviceIdHere]/instantenergy"
    unit_of_measurement: "W"
    value_template: '{{ value_json.instant|int }}'
```

Apologies for my complete lack of Golang knowledge!

timbow committed:
Add power usage reporting

fancygaphtrn commented:
FWIW, thanks for this post.   I never knew how to decode the power settings.

I created a python version of this concept, some time ago and added power to it based on your info, this morning.
https://github.com/fancygaphtrn/voltson-proxy

I created the python version just because I was wanting to learn it some time ago.

brilthor commented:
@bcheck I see this is about a year old now, has the patch been working reliably for you?

I also notice this is still marked as work in progress, is that correct, if so why?

bcheck commented:
Sorry, I switched to using the python version @fancygaphtrn wrote and dropped this.  I can't even exactly remember why, I think the docker version wasn't working on arm (raspbi) for some reason.  I'm going to close this pr anyways.
